app/routes/not-found.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class NotFoundRoute extends Route {}
```

Comment:
I didn't know this till today. I had a hunch that we would need to return an actual status code of 404 to the user. Since this project uses FastBoot, I think this is how we do so:

```suggestion
import Route from '@ember/routing/route';
import { inject as service } from '@ember/service';

export default class NotFoundRoute extends Route {
  @service fastboot;

  beforeModel() {
    if (!this.fastboot.isFastBoot) {
      return;
    }

    this.fastboot.response.statusCode = 404;
  }
}
```

Reply:
If you open Chrome's Network tab, you will see the following.

Before:

<img width="480" alt="Screenshot 2021-03-06 at 08 15 52" src="https://user-images.githubusercontent.com/16869656/110198671-775f0280-7e54-11eb-919c-9559454aa5fb.png">

After:

<img width="480" alt="Screenshot 2021-03-06 at 08 20 26" src="https://user-images.githubusercontent.com/16869656/110198722-d3c22200-7e54-11eb-9e7e-0416f90afefc.png">

(If you click on Chrome's Console tab, you will now see the error message `GET http://localhost:4200/foo-bar-baz 404 (Not Found)` as well.)

I learned the information from:

- https://ember-fastboot.com/docs/user-guide#status-code
- https://discuss.emberjs.com/t/404-when-route-is-known-but-model-is-missing

Reply:
TIL 😄